MdePkg/Test/Mock/Library/GoogleTest/Protocol/MockAcpiSystemDescriptionTable.cpp

```diff
@@ -0,0 +1,36 @@
+/** @file MockAcpiSystemDescriptionTable.cpp

```

Comment:
Please remove all the protocol cpp files as in this PR: https://github.com/microsoft/mu_basecore/pull/1111/files/17a677c2f628b17a077ee61739cd249208ee0b1a#r1727583559